<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPOptionDialog</class>
 <widget class="QDialog" name="DPOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>246</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>246</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Option Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Epsilon:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Layering threshold:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxLayeringThreshold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEpsilon">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap-to-vertex threshold:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSnapVertexThreshold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap-to-edge threshold:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSnapEdgeThreshold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
